Exception thrown on bucket create with Smart Flank feature

## Problem
When gc bucket doesn't exists and Flank tries to create new one - crash happens

<details>
<summary>Click to expand Stacktrace</summary>
Caused by: java.lang.NullPointerException
    at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:882)
    at com.google.cloud.storage.BucketInfo$BuilderImpl.build(BucketInfo.java:649)
    at ftl.args.ArgsHelper.createGcsBucket(ArgsHelper.kt:128)
    at ftl.args.ArgsHelper.createJunitBucket(ArgsHelper.kt:103)
    at ftl.args.AndroidArgs.<init>(AndroidArgs.kt:87)
    at ftl.args.AndroidArgs$Companion.load(AndroidArgs.kt:180)
    at ftl.args.AndroidArgs$Companion.load(AndroidArgs.kt:173)
    at ftl.cli.firebase.test.android.AndroidRunCommand.run(AndroidRunCommand.kt:33)
    at picocli.CommandLine.execute(CommandLine.java:1043)
    ... 9 more
</details>

## Cause
Seems that regression after SmartFlank commit in file [ArgsHelper.kt](
https://github.com/TestArmada/flank/blob/master/test_runner/src/main/kotlin/ftl/args/ArgsHelper.kt#L124).
`targetBucket` is always `null` here. Should be `bucket`

P.S. I wanted to to contrib fix myself, but tests doesn't work by default after repo clone :crying_cat_face:

--- a/release_notes.md
+++ b/release_notes.md
@@ -1,6 +1,6 @@
 ## v4.2.0 (unreleased)
 
-- TODO
+- Fix create Gcs bucket [#444](https://github.com/TestArmada/flank/pull/444)
 
 ## v4.1.1
 

--- a/test_runner/src/main/kotlin/ftl/args/ArgsHelper.kt
+++ b/test_runner/src/main/kotlin/ftl/args/ArgsHelper.kt
@@ -121,7 +121,7 @@ object ArgsHelper {
         if (targetBucket != null) return targetBucket
 
         return storage.create(
-            BucketInfo.newBuilder(targetBucket)
+            BucketInfo.newBuilder(bucket)
                 .setStorageClass(StorageClass.REGIONAL)
                 .setLocation(storageLocation)
                 .setLabels(bucketLabel)

--- a/test_runner/src/main/kotlin/ftl/args/ArgsHelper.kt
+++ b/test_runner/src/main/kotlin/ftl/args/ArgsHelper.kt
@@ -117,8 +117,7 @@ object ArgsHelper {
 
         val bucketListOption = Storage.BucketListOption.prefix(bucket)
         val storageList = storage.list(bucketListOption).values?.map { it.name } ?: emptyList()
-        val targetBucket = storageList.find { it == bucket }
-        if (targetBucket != null) return targetBucket
+        if (storageList.contains(bucket)) return bucket
 
         return storage.create(
             BucketInfo.newBuilder(bucket)

